ee/clickhouse/queries/retention/retention_actors.py

```diff
@@ -94,7 +94,7 @@ def actors(self):
                     for interval_number in range(self._filter.total_intervals - (self._filter.selected_interval or 0))
                 ],
             }
-            for actor in sorted(actor_appearances, key=lambda x: x.actor_id)
+            for actor in sorted(actor_appearances, key=lambda x: x.appearance_count, reverse=True)
```

Comment:
non-blocking

fyi this won't work with pagination, i.e. the person with the highest count could be on page 2 I think